### PR TITLE
Luminate Pipeline Config

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

File: configurations/luminate_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,257 @@
+from core_data_modules.cleaners import swahili
+from src.pipeline_configuration_spec import *
+
+PIPELINE_CONFIGURATION = PipelineConfiguration(
+    pipeline_name="Luminate",
```
Comment (as2388) [inline configurations/luminate_pipeline.py:5]: Is it worth calling this Luminate-S01 or is this likely to be the only series?

File: configurations/luminate_pipeline.py [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+                    FlowResultConfiguration("Luminate_s01_demogs", "pool_kenya_disabled", "disabled"),
+                    FlowResultConfiguration("Luminate_s01_demogs", "preferred_language", "preferred_language"),
+
+                    FlowResultConfiguration("Luminate_s01e01_activation", "rqa_s01e01", "luminate_s01e01")
+                ],
+            )
+        )
+    ],
+    google_form_sources=[],
+    kobotoolbox_sources=[],
```
Comment (as2388) [inline configurations/luminate_pipeline.py:52]: Consider deleting these lines for simplicity

File: configurations/docker_image_name.txt [hunk 3]
```diff
@@ -1 +1 @@
-ghcr.io/africasvoices/engagement-data-pipeline:handle-whatsapp-urns-rapid-pro-to-engagement-db-1
+ghcr.io/africasvoices/engagement-data-pipeline:handle-whatsapp-urns-rapid-pro-to-engagement-db-2
```
Comment (as2388) [inline configurations/docker_image_name.txt:1]: What's the status of a PR and deployment of this branch? I think using WhatsApp might need some special care in the pipelines so we need to be careful not to use it before it's ready.

**Review by as2388 — APPROVED** [review]

File: configurations/luminate_pipeline.py (hunk 2 above)
Comment (DanielMwendwa) [reply configurations/luminate_pipeline.py]: updated, thanks.

File: configurations/luminate_pipeline.py (hunk 1 above)
Comment (DanielMwendwa) [reply configurations/luminate_pipeline.py:5]: Their might be another series, renamed to Luminate-S01 for now, thnaks.

File: configurations/docker_image_name.txt (hunk 3 above)
Comment (DanielMwendwa) [reply configurations/docker_image_name.txt:1]: Its ready for deployment. Discussed offline with DT.

Comment (as2388) [reply configurations/docker_image_name.txt:1]: In that case can you release and use the released version?

**Pull request merged**